docs/exposing_dns.md

```diff
@@ -40,18 +40,24 @@ kubectl create -n ingress-nginx cm udp-services --from-literal="53"="k8gb/k8gb-c
 
 [Local project setup](./local.md) does this patching automatically.
 
+## CoreDNS service lookup
+k8gb controller will first try to find CoreDNS, by looking into a namespace where controller is deployed and find service which annotated with `app.kubernetes.io/name=coredns`. If service have `Status.LoadBalancer.Ingress[0].Hostname` k8gb will use those IPs to reach CoreDNS on this cluster. In case service is not found or have `Status.Loadbalancer.Ingress[0].Hostname` empty, k8gb controller will fallback into Ingress IP lookup.
```

Comment:
```suggestion
k8gb controller will first try to find CoreDNS, by looking into a namespace where controller is deployed and finding a service annotated with `app.kubernetes.io/name=coredns`. If service has `Status.LoadBalancer.Ingress[0].Hostname`, k8gb will use the resolved IPs to reach CoreDNS on this cluster. In case is service is not found or its `Status.Loadbalancer.Ingress[0].Hostname` is empty, k8gb controller will fallback into Ingress IP lookup.
```

Reply:
@k0da this suggestion doesn't look like resolved too.